archive/orphan-configs/CLAUDE.md

```
@@ -0,0 +1,3 @@
+<claude-mem-context>
+
+</claude-mem-context>
```

markdownlint-cli2 (0.22.1)

[warning] Files should end with a single newline character

(MD047, single-trailing-newline)